packages/expo/src/environment/DevAppContainer.tsx

```diff
@@ -2,13 +2,11 @@ import * as React from 'react';
 
 import DevLoadingView from '../environment/DevLoadingView';
 
-export default class DevAppContainer extends React.Component<{ children?: React.ReactNode }> {
-  render() {
-    return (
-      <>
-        {this.props.children}
-        <DevLoadingView />
-      </>
-    );
-  }
+export default function DevAppContainer({ children }: { children?: React.ReactNode }) {
```

Comment:
```suggestion
export default function DevAppContainer({ children }: React.PropsWithChildren<object>) {
```

Let's not type `children` manually.